storage/storage.go

```diff
@@ -105,41 +106,48 @@ type Client struct {
 func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error) {
 	var host, readHost, scheme string
 
+	// In general, it is recommended to use NewService instead of NewClient, since NewService
```

Comment:
Make this `raw.NewService` etc. to make it clear what's being referred to here?

Reply:
done.